Dockerfile

```diff
@@ -1,14 +1,21 @@
 FROM alpine:3.14 AS builder
 
 RUN apk add --no-cache tini gcc g++ pkgconf make \
-	util-linux-dev sqlite-dev mariadb-connector-c-dev cmake zlib-dev fmt-dev \
-	file-dev libexif-dev curl-dev ffmpeg-dev ffmpegthumbnailer-dev \
-	libmatroska-dev libebml-dev taglib-dev pugixml-dev spdlog-dev \
-	duktape-dev libupnp-dev git bash
+    util-linux-dev sqlite-dev mariadb-connector-c-dev cmake zlib-dev fmt-dev \
+    file-dev libexif-dev curl-dev ffmpeg-dev ffmpegthumbnailer-dev \
+    libmatroska-dev libebml-dev taglib-dev pugixml-dev spdlog-dev \
+    duktape-dev git bash \
+    # to build libupnp
+    autoconf automake libtool file
 
 WORKDIR /gerbera_build
 
 COPY . .
+
+# Build libupnp
+RUN scripts/install-pupnp.sh
```

Comment:
You should copy just the script here, so that layer caching works if there are no changes to the file.

As it is this will cause pupnp to rebuilt every time.

Reply:
done